docroot/modules/custom/sitenow_p2lb/sitenow_p2lb.module

```diff
@@ -1057,8 +1057,7 @@ function sitenow_p2lb_section_image($section_image_fid, $banner_text, $node) {
   // Assumes only one button or link is present.
   // Any more links will remain in the body.
   if (preg_match('|(.*?)(<a.*?href="(.*?)".*?>(.*?)<\/a>)(.*)|is', $text['value'], $matches)) {
-    // Attach the bookended content into a single value.
-    $text['value'] = strip_tags($matches[1] . $matches[5]);
+    $text['value'] = $matches[1] . $matches[5];
```

Comment:
```suggestion
    $text['value'] = strip_tags($matches[1] . $matches[5], ['p', 'a']);
```

>as of PHP 7.4.0 the line above can be written as: echo strip_tags($text, ['p', 'a']);

not tested

Reply:
Thanks @joewhitsitt, I think we want something more like `$text['value'] = strip_tags($matches[1] . $matches[5], ['p', 'a', 'strong', 'em', 'h2', 'h3', 'h4','h6' ]);` that we can update as we go.  How does that sound?  I can post example screenshots if you'd like to see some samples.

Reply:
updated the commit suggestion based on your comment. again, i haven't tested anything yet.